app/models/admin_menu.rb

```diff
@@ -40,7 +40,6 @@ class AdminMenu
       item(name: "mods"),
       item(name: "moderator actions ads", controller: "moderator_actions"),
       item(name: "privileged reactions"),
-      # item(name: "interaction limits")
```

Comment:
This doesn't appear to be useful, so I've decided to remove it. Please let me know if I am missing something, though!